app/context-map/workload.js

```diff
@@ -3,14 +3,14 @@ const mapToTiers = require('./tiers')
 const Locations = require('../../app/staging/constants/locations')
 const assertArrayLength = require('../../app/points/domain/validation/assert-array-length')
 
-module.exports = function (communityTierCounts, custodyTierCounts, licenseTierCounts) {
+module.exports = function (communityTierCounts, custodyTierCounts, licenseTierCounts, workloadId) {
```

Comment:
I don't think we'll have the workloadId until it is written to the database so this can be removed from the context mapper.